docs/docs/reference-events.md

```diff
@@ -338,3 +338,12 @@ string pseudoElement
 float elapsedTime
 ```
 
+* * *
+
+### Misc Events
```

Comment:
Not sure where to add this event on the docs. Does not feel right in any of the current categories, that's why I added a new one, but still not sure is the way to go